### DISCUSSION TITLE
help needed in local development setup

### DISCUSSION THREAD

**Discussion by itskunalsingh** [outline/outline#2972] [Self Hosting]
whenever i run "make up " i get these errors:

Error: Cannot find module 'webpack-cli/package.json'

C:\Users\KUNAL SINGH\Outline\outline>babel --extensions .ts,.tsx --quiet -d ./build/server ./server && babel --quiet --extensions .ts,.tsx -d./build/shared ./shared && 
cp ./server/collaboration/Procfile ./build/server/collaboration/Procfile && cp package.json ./build && ln -sf \"$(pwd)/webpack.config.dev.js\" ./build
'babel' is not recognized as an internal or external command,
operable program or batch file.

**Comment by tommoor** [comment]
I'm not sure, it could be a Windows thing? The maintainer team only develops on unix machines.

**Reply by itskunalsingh** [reply]
I am also facing issue while running it using docker? Can you suggest something regarding this ?

**Reply by itskunalsingh** [reply]
Postgres is running on port 5432 but when i check localhost for that it doesn't show anything.

**Comment by itskunalsingh** [comment]
![Screenshot (1)](https://user-images.githubusercontent.com/39680932/151335542-0c6fdb2c-2b24-4f3a-b507-30f038b4d7a1.png)
Can you review this screenshot and help in resolving the issue.

**Reply by tommoor** [reply]
It's not working because symbolic links don't exist on Windows. If could probably change to a file copy operation or maybe there is some other equivalent on windows that I'm not aware of.

The ln -s command

**Reply by itskunalsingh** [reply]
Ok, can you find some solutions if i want to run it on windows OS. If you find something for this please let me know, it will be helpful.
